detox/test/android/app/build.gradle

```diff
@@ -61,19 +71,10 @@ android {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro', "${project(':detox').projectDir}/proguard-rules-app.pro"
-
+            testProguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro', "${project(':detox').projectDir}/proguard-rules-app.pro"
```

Comment:
This doesn't make a whole lot of sense. `proguard-rules-app.pro` is a set of rules meant to be applied for the app's build. For the test, the equivalent file in Detox - `proguard-rules.pro` is published and applied automatically over the test-code build, as it is bundled into the `.aar`. Obviously, since this is a Detox-compiling project, the .aar is not part of the game, but there ought to be a better way to do this.